pkg/osquery/tables/dataflattentable/exec.go

```diff
@@ -15,21 +15,36 @@ import (
 	"github.com/pkg/errors"
 )
 
-func TablePluginExec(client *osquery.ExtensionManagerClient, logger log.Logger, tableName string, dataSourceType DataSourceType, execArgs []string) *table.Plugin {
+type ExecTableOpt func(*Table)
+
+func WithSeperator(separator string) ExecTableOpt {
```

Comment:
```suggestion
// WithSeparator sets the field separator. If not specified, it defaults to `:`
func WithSeparator(separator string) ExecTableOpt {
```

Mostly wanted to note that it's the _field_ separator, note that it has a default, and fix your spelling error